sam_bot_description/src/description/sam_bot_description.urdf

```diff
@@ -222,31 +216,33 @@
   </gazebo>
 
   <gazebo>
-    <plugin name='diff_drive' filename='libgazebo_ros_diff_drive.so'>
-      <ros>
-        <namespace>/demo</namespace>
-        <remap from="cmd_vel" to="/cmd_vel"/>
-      </ros>
-
+    <plugin filename="gz-sim-diff-drive-system" name="gz::sim::systems::DiffDrive">
```

Comment:
I believe this is in the SDF file now, right?

Reply:
Yes, it's in both the URDF and SDF.